experimental/traffic-portal/src/app/api/testing/server.service.ts

```diff
@@ -302,4 +303,55 @@ export class ServerService {
 		srv.statusId = status.id;
 		srv.offlineReason = offlineReason ?? null;
 	}
+
+	/**
+	 * Creates a status.
+	 *
+	 * @param status The status details (name & description) to create. Description is an optional property in status.
+	 * @returns The status as created and returned by the API.
+	 */
+	public async createStatus(status: RequestStatus): Promise<ResponseStatus> {
+		const newStatus = {
+			description: status.description ? status.description : null,
+			id: ++this.statusIdCounter,
+			lastUpdated: new Date(),
+			name: status.name
+		};
+		this.statuses.push(newStatus);
+		return newStatus;
+	}
+
+	/**
+	 * Updates status Details.
+	 *
+	 * @param payload containes name and description for the status., unique identifier thereof.
+	 * @param id The Status ID
+	 */
+	public async updateStatusDetail(payload: ResponseStatus, id: number): Promise<ResponseStatus> {
```

Comment:
This testing service method's call signature is not assignable to the "concrete" service method's signature, but in order for the testing services to work properly as drop-in replacements for their "concrete" counterparts, their call signatures must match.